scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -930,4 +949,327 @@ <h4 class="modal-title">Edit synopsis</h4>
+                             'cytoband.txt.chr11.png', 'cytoband.txt.chr12.png',
+                             'cytoband.txt.chr13.png', 'cytoband.txt.chr14.png',
+                             'cytoband.txt.chr15.png', 'cytoband.txt.chr16.png',
+                             'cytoband.txt.chr17.png', 'cytoband.txt.chr18.png',
+                             'cytoband.txt.chr19.png', 'cytoband.txt.chr20.png',
+                             'cytoband.txt.chr21.png', 'cytoband.txt.chr22.png',
+                             'cytoband.txt.chrX.png', 'cytoband.txt.chrY.png'];
+
+    const ROH_IMAGES = ['roh_chr1.png', 'roh_chr2.png', 'roh_chr3.png',
+                        'roh_chr4.png', 'roh_chr5.png', 'roh_chr6.png',
+                        'roh_chr7.png', 'roh_chr8.png', 'roh_chr9.png',
+                        'roh_chr10.png', 'roh_chr11.png', 'roh_chr12.png',
+                        'roh_chr13.png', 'roh_chr14.png', 'roh_chr15.png',
+                        'roh_chr16.png', 'roh_chr17.png', 'roh_chr18.png',
+                        'roh_chr19.png', 'roh_chr20.png', 'roh_chr21.png',
+                        'roh_chr22.png', 'roh_chrX.png', 'roh_chrY.png'];
+
+    const UPD_IMAGES = ['upd_chr1.png', 'upd_chr2.png', 'upd_chr3.png',
+                        'upd_chr4.png', 'upd_chr5.png', 'upd_chr6.png', 
+                        'upd_chr7.png', 'upd_chr8.png', 'upd_chr9.png',
+                        'upd_chr10.png', 'upd_chr11.png', 'upd_chr12.png',
+                        'upd_chr13.png', 'upd_chr14.png', 'upd_chr15.png',
+                        'upd_chr16.png', 'upd_chr17.png', 'upd_chr18.png',
+                        'upd_chr19.png', 'upd_chr20.png', 'upd_chr21.png',
+                        'upd_chr22.png', 'upd_chrX.png', 'upd_chrY.png'];
+
+
+    // TODO: Handle trisomy
+    const CHROMOSOMES = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10',
+                         '11', '12', '13', '14', '15', '16', '17', '18',
+                         '19', '20', '21', '22', 'X',' Y'];
+
+
+
+
+    const NUMBER_OF_COLUMNS = 3        
+    const OFFSET_X = 60;
+    const OFFSET_Y = 30;
+
+    // skära ut kromosom
+    // "M 300 150 L 300 500 L 500 500 A 50 50 0 1 1 500 150 L 300 150"
+
+    
+
+    // Draw ROH call pictures -coverage- and UPOD pictures -color coded
+    // genome regions- onto the dashboard.
+    //
+    //
+    function draw_tracks(){
+        const CYT_HEIGHT = 50 ;
+        const CYT_WIDTH = 500 ;
+        const ROH_IMGPATH = '/public/static/roh_images/'
+        const IDEO_IMGPATH = '/public/static/chr_images/'
+        
+        console.log("draw roh") 
+        var roh_svg = document.getElementById("roh_svg");
+
+        var roh_imgObj = new Image();
+        
+        var upd_imgPath = '/public/static/upd_images/'
+        var upd_imgObj = new Image();
+        // Loop through to add ROH and UPD images
+
+        var chrom1 = {name: '1', length: 482, cent_start: 245, cent_length: 13 }
```

Comment:
So, I guess these could stay state until we get a hg38 - but given time we should prepare for at least two different versions